applications/tari_base_node/src/command_handler.rs

```diff
@@ -116,7 +116,8 @@ impl CommandHandler {
             let mut status_line = StatusLine::new();
             let version = format!("v{}", consts::APP_VERSION_NUMBER);
             status_line.add_field("", version);
-
+            let network = format!("{}", config.network);
+            status_line.add_field("", network);
```

Comment:
incredibly minor nit but `add_field` can take anything that implements `ToString` so you could just do this I think

```suggestion
            status_line.add_field("", config.network);
```